cunumeric/_ufunc/ufunc.py

```diff
@@ -203,6 +264,14 @@ def __call__(
     ):
         x = convert_to_cunumeric_ndarray(x)
 
+        if len(args) > 0:
```

Comment:
Was this ever an issue? It doesn't seem possible in regular python to pass the same argument as both a positional and a keyword argument:
```
def foo(a, b, out=None):
    print(a)
    print(b)
    print(out)

foo(1,2,3,out=4)
```

```
~> python test.py
Traceback (most recent call last):
  File "test.py", line 6, in <module>
    foo(1,2,3,out=4)
TypeError: foo() got multiple values for argument 'out'
```

Reply:
I believe NumPy's ufunc has the very same issue and the exception I'm raising [here](https://github.com/nv-legate/cunumeric/pull/315/files/ab6b1fc5bfd343a0c1281e43f1a8a238ef1f77f4#diff-dc4602a545cbb84b22aff9dcc1447e6763980e9d69af1abe1d7d9966d099c470R269) is consistent with NumPy's behavior. Note that ufuncs (more precisely, the template for ufuncs) actually take a variable number of arguments and do the validity check based on their arity. So, it's not reasonable that their behavior deviates from the normal Python behavior.